src/Compiler/Reports/AbstractReportManager.cpp

```diff
@@ -207,6 +211,36 @@ TaskMessage AbstractReportManager::createWarningErrorItem(
 
 void AbstractReportManager::fillTimingData(const QStringList &timingData) {
   m_timingData.clear();
+
+  createTimingDataFile(timingData);
+
+  auto timingStr = timingData.join(' ');
+  auto matchIt = FIND_STAT_TIMING.globalMatch(timingStr);
+  if (FIND_STAT_TIMING.captureCount() != TIMING_FIELDS.size()) return;
+  auto valueIndex = 0;
+  while (matchIt.hasNext()) {
+    auto match = matchIt.next();
+    m_timingData.push_back({TIMING_FIELDS[valueIndex++], match.captured()});
+  }
+}
+
+void AbstractReportManager::createTimingDataFile(
+    const QStringList &timingData) {
+  auto logFileName = getTimingLogFileName();
+  if (logFileName.isEmpty()) return;
+
+  auto projectPath = Project::Instance()->projectPath();
+  auto logFilePath = QString("%1/%2").arg(projectPath, logFileName);
```

Comment:
Converting to qstring while creating the path might not be safe on windows.
For these we try to use the std::filesystem so that the / operator will hopefully provide a path that works in both environments.
Example of filesystem path creation: https://github.com/os-fpga/FOEDAG/blob/da0771b6c16f717e0462d16e8fd32e515d0c6835/src/Compiler/Compiler.cpp/#L2242-L2243

Reply:
I just copied that code from another place (I'm an author anyway :) ). fixed